src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -85,19 +65,27 @@ const RadioButtonGroup = (props) => {
     );
   });
 
-  const icon = () => {
-    const type = getValidationType(props);
-    const message = getValidationMessage(type, props);
-
+  const type = getValidationType(props);
+  const labelWithValidationIcon = () => {
     return (
-      <ValidationIconStyle type={ type }>
-        <Icon
-          type={ type }
-          tooltipMessage={ message }
-        />
-      </ValidationIconStyle>
+      <React.Fragment>
```

Comment:
it could be just `<>` instead of `<React.Fragment>`

Reply:
This will become redundant with the changes from PR FE-1728